Invalid invocation: chaincode \'daml_on_fabric\' has not been initialized for this version, must call as init

Versions:
Java: 1.8.0
Scala: 2.12.7 (and 2.12.11)
SBT: 1.2.8

Steps to reproduce: 

[Run through Katacoda scenario as here](https://www.katacoda.com/anthonylusardi-da/courses/deploying-daml/daml-on-fabric)

Once you get to the step where you run `sbt "run --port 6865 --role provision,time,ledger" -J-DfabricConfigFile=config-local.yaml` you'll get the error pasted below.

I've tried adding `--isInit` to the `sbt` command as [noted here](https://stackoverflow.com/questions/60391417/error-in-committed-chaincode-invoke-query-with-hyperledger-fabric-2-0) for the `peer` command but it doesn't help.

```
[info]  [SUCCESSFUL ] org.scala-lang#scala-compiler;2.12.11!scala-compiler.jar (342ms)
[info] Done updating.
[warn] There may be incompatibilities among your library dependencies; run 'evicted' to see detailed eviction warnings.
[info] Compiling 10 Scala sources and 7 Java sources to /root/daml-on-fabric/target/scala-2.12/classes ...
[info] Non-compiled module 'compiler-bridge_2.12' for Scala 2.12.11. Compiling...
[info]   Compilation completed in 8.87s.
[info] Done compiling.
[warn] Multiple main classes detected.  Run 'show discoveredMainClasses' to see the list
[info] Packaging /root/daml-on-fabric/target/scala-2.12/daml-on-fabric_2.12-2.0.0.jar ...
[info] Done packaging.
[info] Running com.daml.DamlOnFabricServer --port 6865 --role provision,time,ledger
log4j:WARN No appenders could be found for logger (org.hyperledger.fabric.sdk.helper.Config).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
Admin@org1.example.com private key: /root/daml-on-fabric/src/test/fixture/tmp/data/crypto-config/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/keystore/priv_sk
Admin@org1.example.com sign cert: /root/daml-on-fabric/src/test/fixture/tmp/data/crypto-config/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/signcerts/Admin@org1.example.com-cert.pem
Admin@org1.example.com private key: /root/daml-on-fabric/src/test/fixture/tmp/data/crypto-config/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/keystore/priv_sk
Admin@org1.example.com sign cert: /root/daml-on-fabric/src/test/fixture/tmp/data/crypto-config/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/signcerts/Admin@org1.example.com-cert.pem
Admin@org2.example.com private key: /root/daml-on-fabric/src/test/fixture/tmp/data/crypto-config/peerOrganizations/org2.example.com/users/Admin@org2.example.com/msp/keystore/priv_sk
Admin@org2.example.com sign cert: /root/daml-on-fabric/src/test/fixture/tmp/data/crypto-config/peerOrganizations/org2.example.com/users/Admin@org2.example.com/msp/signcerts/Admin@org2.example.com-cert.pem
Constructing channel mainchannel
Admin@org1.example.com private key: /root/daml-on-fabric/src/test/fixture/tmp/data/crypto-config/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/keystore/priv_sk
Admin@org1.example.com sign cert: /root/daml-on-fabric/src/test/fixture/tmp/data/crypto-config/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/signcerts/Admin@org1.example.com-cert.pem
Created channel mainchannel
Admin@org1.example.com private key: /root/daml-on-fabric/src/test/fixture/tmp/data/crypto-config/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/keystore/priv_sk
Admin@org1.example.com sign cert: /root/daml-on-fabric/src/test/fixture/tmp/data/crypto-config/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/signcerts/Admin@org1.example.com-cert.pem
Peer peer0 grpcs://localhost:10051 joined channel mainchannel
Admin@org2.example.com private key: /root/daml-on-fabric/src/test/fixture/tmp/data/crypto-config/peerOrganizations/org2.example.com/users/Admin@org2.example.com/msp/keystore/priv_sk
Admin@org2.example.com sign cert: /root/daml-on-fabric/src/test/fixture/tmp/data/crypto-config/peerOrganizations/org2.example.com/users/Admin@org2.example.com/msp/signcerts/Admin@org2.example.com-cert.pem
Peer peer0 grpcs://localhost:10056 joined channel mainchannel
Joined peers to channel mainchannel
No META-INF folder detected. Starting chaincode without any custom indexes
Orgs installs the chaincode on its peers.
Admin@org1.example.com private key: /root/daml-on-fabric/src/test/fixture/tmp/data/crypto-config/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/keystore/priv_sk
Admin@org1.example.com sign cert: /root/daml-on-fabric/src/test/fixture/tmp/data/crypto-config/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/signcerts/Admin@org1.example.com-cert.pem
Getting chaincode packageID status SUCCESS and packageID daml_on_fabric:4cd11263a3097bcfe2e3c5c66c36a9e54774e39bb7e3ea619a3549cc4ba8c799
Successful install proposal response Txid: f6b4749236d02a89f072ff8550c503b989c61f1f81e12167f5f92fb3a15fe768 from peer peer0
Received 1 install proposal responses. Successful+verified: 1 . Failed: 0
verifyByQueryInstalledChaincode: Peer peer0 with status SUCCESS , label daml_on_fabric
Org going to approve chaincode definition for my org.
lifecycleApproveChaincodeDefinitionForMyOrg: Peer grpcs://localhost:10051 status SUCCESS verified true
Checking on org's network for approvals
verifyByCheckCommitReadinessStatus Peer grpcs://localhost:10051 Status SUCCESS Approved Orgs [[Org1MSP]] UnApproved Orgs [[Org2MSP]]
verifyByCheckCommitReadinessStatus Peer grpcs://localhost:10056 Status SUCCESS Approved Orgs [[Org1MSP]] UnApproved Orgs [[Org2MSP]]
Admin@org2.example.com private key: /root/daml-on-fabric/src/test/fixture/tmp/data/crypto-config/peerOrganizations/org2.example.com/users/Admin@org2.example.com/msp/keystore/priv_sk
Admin@org2.example.com sign cert: /root/daml-on-fabric/src/test/fixture/tmp/data/crypto-config/peerOrganizations/org2.example.com/users/Admin@org2.example.com/msp/signcerts/Admin@org2.example.com-cert.pem
Getting chaincode packageID status SUCCESS and packageID daml_on_fabric:4cd11263a3097bcfe2e3c5c66c36a9e54774e39bb7e3ea619a3549cc4ba8c799
Successful install proposal response Txid: b7fbe11258d0b4dbaf53b07c72964e2f814ecf2c5c240e9b2cac21a7061321dc from peer peer0
Received 1 install proposal responses. Successful+verified: 1 . Failed: 0
verifyByQueryInstalledChaincode: Peer peer0 with status SUCCESS , label daml_on_fabric
Org going to approve chaincode definition for my org.
lifecycleApproveChaincodeDefinitionForMyOrg: Peer grpcs://localhost:10056 status SUCCESS verified true
Checking on org's network for approvals
verifyByCheckCommitReadinessStatus Peer grpcs://localhost:10051 Status SUCCESS Approved Orgs [[Org2MSP, Org1MSP]] UnApproved Orgs [[]]
verifyByCheckCommitReadinessStatus Peer grpcs://localhost:10056 Status SUCCESS Approved Orgs [[Org2MSP, Org1MSP]] UnApproved Orgs [[]]
Org doing commit chaincode definition
commitChaincodeDefinitionRequest response: Peer grpcs://localhost:10051 Status SUCCESS Verified true
commitChaincodeDefinitionRequest response: Peer grpcs://localhost:10056 Status SUCCESS Verified true
Invoke chaincode - going to call: init on the chaincode daml_on_fabric
Received 2 tx proposal responses. Successful+verified: 0 . Failed: 2  - Fcn: init
Not enough endorsers for executeChaincode(move a,b,100):2 endorser error: error in simulation: failed to execute transaction 8fabc8504ee7a6a742a6a879f0de2d53b70ff37702858d2ab130066ffac41324: could not launch chaincode daml_on_fabric:4cd11263a3097bcfe2e3c5c66c36a9e54774e39bb7e3ea619a3549cc4ba8c799: chaincode registration failed: error starting container: error starting container: API error (404): network daml-on-fabric_default not found. Was verified: false
Caught an exception running Channel mainchannel
java.util.concurrent.CompletionException: org.hyperledger.fabric.sdk.exception.InvalidArgumentException: proposalResponses collection is empty
        at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
        at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
        at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:783)
        at java.util.concurrent.CompletableFuture.uniWhenCompleteStage(CompletableFuture.java:792)
        at java.util.concurrent.CompletableFuture.whenComplete(CompletableFuture.java:2153)
        at org.hyperledger.fabric.sdk.Channel.sendTransaction(Channel.java:5534)
        at org.hyperledger.fabric.sdk.Channel.sendTransaction(Channel.java:5236)
        at org.hyperledger.fabric.sdk.Channel.sendTransaction(Channel.java:5021)
        at org.hyperledger.fabric.sdk.Channel.sendTransaction(Channel.java:5009)
        at com.daml.FabricContext.invokeChaincode(FabricContext.java:695)
        at com.daml.FabricContext.invokeChaincode(FabricContext.java:631)
        at com.daml.FabricContext.invokeChaincode(FabricContext.java:627)
        at com.daml.FabricContext.runChannel(FabricContext.java:288)
        at com.daml.FabricContext.initNetworkConfiguration(FabricContext.java:169)
        at com.daml.FabricContext.<init>(FabricContext.java:103)
        at com.daml.DAMLKVConnector.<init>(DAMLKVConnector.java:74)
        at com.daml.DAMLKVConnector.get(DAMLKVConnector.java:30)
        at com.daml.DamlOnFabricServer$.delayedEndpoint$com$daml$DamlOnFabricServer$1(DamlOnFabricServer.scala:61)
        at com.daml.DamlOnFabricServer$delayedInit$body.apply(DamlOnFabricServer.scala:44)
        at scala.Function0.apply$mcV$sp(Function0.scala:39)
        at scala.Function0.apply$mcV$sp$(Function0.scala:39)
        at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)
        at scala.App.$anonfun$main$1$adapted(App.scala:80)
        at scala.collection.immutable.List.foreach(List.scala:392)
        at scala.App.main(App.scala:80)
        at scala.App.main$(App.scala:78)
        at com.daml.DamlOnFabricServer$.main(DamlOnFabricServer.scala:44)
        at com.daml.DamlOnFabricServer.main(DamlOnFabricServer.scala)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at sbt.Run.invokeMain(Run.scala:98)
        at sbt.Run.run0(Run.scala:92)
        at sbt.Run.execute$1(Run.scala:68)
        at sbt.Run.$anonfun$run$4(Run.scala:80)
        at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
        at sbt.util.InterfaceUtil$$anon$1.get(InterfaceUtil.scala:10)
        at sbt.TrapExit$App.run(TrapExit.scala:253)
        at java.lang.Thread.run(Thread.java:748)
Caused by: org.hyperledger.fabric.sdk.exception.InvalidArgumentException: proposalResponses collection is empty
        at org.hyperledger.fabric.sdk.SDKUtils.getProposalConsistencySets(SDKUtils.java:118)
        at org.hyperledger.fabric.sdk.Channel.doSendTransaction(Channel.java:5571)
        at org.hyperledger.fabric.sdk.Channel.sendTransaction(Channel.java:5533)
        ... 34 more
Admin@org1.example.com private key: /root/daml-on-fabric/src/test/fixture/tmp/data/crypto-config/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/keystore/priv_sk
Admin@org1.example.com sign cert: /root/daml-on-fabric/src/test/fixture/tmp/data/crypto-config/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/signcerts/Admin@org1.example.com-cert.pem
Admin@org2.example.com private key: /root/daml-on-fabric/src/test/fixture/tmp/data/crypto-config/peerOrganizations/org2.example.com/users/Admin@org2.example.com/msp/keystore/priv_sk
Admin@org2.example.com sign cert: /root/daml-on-fabric/src/test/fixture/tmp/data/crypto-config/peerOrganizations/org2.example.com/users/Admin@org2.example.com/msp/signcerts/Admin@org2.example.com-cert.pem
That's all folks!
Admin@org1.example.com private key: /root/daml-on-fabric/src/test/fixture/tmp/data/crypto-config/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/keystore/priv_sk
Admin@org1.example.com sign cert: /root/daml-on-fabric/src/test/fixture/tmp/data/crypto-config/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/signcerts/Admin@org1.example.com-cert.pem
[error] (run-main-0) com.daml.FabricContextException: response {
[error]   status: 500
[error]   message: "error in simulation: failed to execute transaction 1e1238edc4eb2c232b30e2ad88ef35eb69ae6d8eb414c58df998f2dbea583641: invalid invocation: chaincode \'daml_on_fabric\' has not been initialized for this version, must call as init first"
[error] }
[error] com.daml.FabricContextException: response {
[error]   status: 500
[error]   message: "error in simulation: failed to execute transaction 1e1238edc4eb2c232b30e2ad88ef35eb69ae6d8eb414c58df998f2dbea583641: invalid invocation: chaincode \'daml_on_fabric\' has not been initialized for this version, must call as init first"
[error] }
[error]
[error]         at com.daml.FabricContext.queryChaincode(FabricContext.java:578)
[error]         at com.daml.FabricContext.queryChaincode(FabricContext.java:558)
[error]         at com.daml.FabricContext.queryChaincode(FabricContext.java:554)
[error]         at com.daml.DAMLKVConnector.getLedgerId(DAMLKVConnector.java:316)
[error]         at com.daml.DAMLKVConnector.<init>(DAMLKVConnector.java:90)
[error]         at com.daml.DAMLKVConnector.get(DAMLKVConnector.java:30)
[error]         at com.daml.DamlOnFabricServer$.delayedEndpoint$com$daml$DamlOnFabricServer$1(DamlOnFabricServer.scala:61)
[error]         at com.daml.DamlOnFabricServer$delayedInit$body.apply(DamlOnFabricServer.scala:44)
[error]         at scala.Function0.apply$mcV$sp(Function0.scala:39)
[error]         at scala.Function0.apply$mcV$sp$(Function0.scala:39)
[error]         at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)
[error]         at scala.App.$anonfun$main$1$adapted(App.scala:80)
[error]         at scala.collection.immutable.List.foreach(List.scala:392)
[error]         at scala.App.main(App.scala:80)
[error]         at scala.App.main$(App.scala:78)
[error]         at com.daml.DamlOnFabricServer$.main(DamlOnFabricServer.scala:44)
[error]         at com.daml.DamlOnFabricServer.main(DamlOnFabricServer.scala)
[error]         at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[error]         at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[error]         at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[error]         at java.lang.reflect.Method.invoke(Method.java:498)
[error] Nonzero exit code: 1
[error] (Compile / run) Nonzero exit code: 1
[error] Total time: 72 s, completed 08-Jun-2020 16:20:29
```

--- a/src/test/fixture/fabric.sh
+++ b/src/test/fixture/fabric.sh
@@ -16,7 +16,7 @@ if [[ "${DOCKER_COMPOSE_FILE}" == "" ]]; then
 fi
 
 if [[ "${DOCKER_NETWORK}" == "" ]]; then
-  DOCKER_NETWORK="daml-on-fabric"
+  DOCKER_NETWORK="damlonfabric"
 fi
 
 export ORG_HYPERLEDGER_FABRIC_SDKTEST_VERSION="2.0.0"

--- a/src/test/fixture/peer-base/peer-base.yaml
+++ b/src/test/fixture/peer-base/peer-base.yaml
@@ -12,7 +12,7 @@ services:
       # the following setting starts chaincode containers on the same
       # bridge network as the peers
       # https://docs.docker.com/compose/networking/
-      - CORE_VM_DOCKER_HOSTCONFIG_NETWORKMODE=daml-on-fabric_default
+      - CORE_VM_DOCKER_HOSTCONFIG_NETWORKMODE=damlonfabric_default
 #     - CORE_LOGGING_LEVEL=ERROR
       - CORE_LOGGING_LEVEL=DEBUG
       - CORE_PEER_ENDORSER_ENABLED=true

